src/Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -97,6 +114,11 @@ public SkiaEncoder(ILogger<SkiaEncoder> logger, IApplicationPaths appPaths)
     public IReadOnlyCollection<ImageFormat> SupportedOutputFormats
         => new HashSet<ImageFormat> { ImageFormat.Webp, ImageFormat.Jpg, ImageFormat.Png, ImageFormat.Svg };
 
+    /// <summary>
+    /// Gets the default typeface to use.
+    /// </summary>
+    public static SKTypeface DefaultTypeFace => _typefaces.Last();
```

Comment:
Wouldn't it make more sense to put the default first?

Reply:
When we iterate to find best fonts, we start from the beginning of the typefaces array. However, since we begin with the default font, having it at the beginning of the list would always result in a lookup miss for the first fallback try, which I want to avoid.

Reply:
The thing is, we never had a “preferred font” like what most websites do. What we used to do was simply use the system’s default sans-serif font. If the library title contained a CJK character, we would select the system’s default fonts for CJK characters. This way, we were always using the system’s default fonts and if the title is pure ANSI characters we were explicitly picking the default sans-serif font. I’m keeping this logic in place to ensure that the first CJK fonts don’t override the default sans-serif preference.